docs/contributing/code.md

```diff
@@ -30,6 +30,12 @@ Next, navigate into the curio project in your terminal, and install the required
 go mod download
 ```
 
+Finally, we use [direnv](https://direnv.net/) to manage env vars in development. You can set up your own by using `.envrc.example` as a base. You're unlikely to need to set anything other than `GITHUB_WORKSPACE` for integration tests, so feel free to skip this step or export it manually using:
+
+```
+export GITHUB_WORKSPACE=`pwd`
```

Comment:
if you run `scripts/run_tests.sh`, that will be done automatically

Reply:
https://github.com/Bearer/curio/blob/main/scripts/run_tests.sh

Reply:
Maybe we should update the docs a bit based on this @markmichon ?

Reply:
Hmm. Let's keep this open. I'll add the other page to this PR with a note about running the test script instead of the standard `go test ../.` line.